src/Casts/EnumCast.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace BenSampo\Enum\Casts;
+
+use BenSampo\Enum\Enum;
+use Illuminate\Contracts\Database\Eloquent\CastsAttributes;
+
+class EnumCast implements CastsAttributes
+{
+    /**@var string */
+    private $enumClass;
```

Comment:
```suggestion
    protected $enumClass;
```

I think libraries should be extendable.